code/modules/paperwork/papershredder.dm

```diff
@@ -122,18 +187,3 @@
 		M.drop_from_inventory(src)
 	PoolOrNew(/obj/effect/decal/cleanable/ash,get_turf(src))
 	qdel(src)
-
```

Comment:
Simply re-ordered this code so the type definition precedes its procs.  Looks nicer.